tests/unit/jobserver/commands/test_rap.py

```diff
@@ -202,6 +203,9 @@ def test_rap_status_update_single_job_request(
     assert log_output.entries[-1]["updated_job_ids"] == [job1.id]
     check_job_request_status(job_request.identifier, JobRequestStatus.SUCCEEDED)
 
+    trace = get_trace()
```

Comment:
very minor nit; this returns spans from a trace (I think; although maybe it's spans from all traces, possibly more than one - I get confused with the terms); if it's called `spans`, then I'd find `spans[0]` in the assert below clearer.